### ISSUE TITLE
[Windows terminal>Setting>Profile>Appearance]:Color scheme names are truncated after applying resize settings.

### ISSUE THREAD

**Issue by Saurabh220304** [microsoft/terminal#19698]
### Steps to reproduce

Steps for applying Resize settings:
Settings>Display>Display Resolution (1680*1050)>Set scaling to 100%>Increase the text slider to 200% >Apply>Reopen the application.

Repro Steps:
1.Launch the Window terminal app.
2.Navigate to the 'New Open Tab' and select the 'Settings' menu item.
3.Settings pag will open and Follow all pre-requisites..
4.Navigate to Profile and select default.
5.Default page will open.
6.Navigate to Appearance and select the color sceheme 
7.Observe the issue here.

User Experience:
Users who rely on larger text for readability (low vision, cognitive disabilities) cannot access full chat content after resizing. This creates barriers for communication and understanding, impacting usability and accessibility.

WCAG Reference Link:
https://www.w3.org/WAI/WCAG21/Understanding/resize-text

Attachements:
https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-heparihar_microsoft_com/IQD-vqqzX3ErSKGC3CGntudiAc10F498rtUhr1QZvCYVUkY?e=1Imoer

### Expected Behavior

The color scheme names should remain fully visible after resizing, or an accessible mechanism (e.g., tooltip on hover/focus or expandable view) should be provided to reveal the complete name.

### Actual Behavior

After applying resize settings, color scheme names are truncated and not fully visible. No tooltip or alternative mechanism is available to reveal the complete name.

**Comment by carlos-zamora** [comment]
Hi @Saurabh220304. What is the associated severities for this and the other accessibility bugs you filed? 

As for a fix, I think just adding a tooltip should work. Should be pretty easy.

## Images

<img width="1857" height="650" alt="Image" src="https://github.com/user-attachments/assets/f4c7af95-2ca2-4926-a4ca-2cfd248072ad" />
<img width="1857" height="650" alt="Image" src="https://github.com/user-attachments/assets/db9958fc-9e48-4c8d-9160-5938652652bc" />
